modules/reindex/src/test/java/org/elasticsearch/index/reindex/ReindexFromRemoteWhitelistTests.java

```diff
@@ -37,6 +39,9 @@
  * Tests the reindex-from-remote whitelist of remotes.
  */
 public class ReindexFromRemoteWhitelistTests extends ESTestCase {
+
+    private final BytesReference matchAll = new BytesArray(new MatchAllQueryBuilder().toString());
```

Comment:
I would prefer `new BytesArray("{ \"foo\" : \"bar\" }"`, makes it more explicit what we rely on (JSON and not query-validity).